ledger/simulation/testing/utils.go

```diff
@@ -78,6 +80,7 @@ func (info TxnInfo) InnerTxn(parent transactions.SignedTxn, inner txntest.Txn) t
 
 // Environment contains the ledger and testing environment for transaction simulations
 type Environment struct {
```

Comment:
Could you say a bit more about Environment in the comment.  Is it just a convenient way to build up a particular ledger state, so that you can then do a simulate and get an expected answer?

Reply:
Yes, I'll add to that comment